examples/resource_lacework_report_rule/main.tf

```diff
@@ -0,0 +1,101 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+resource "lacework_report_rule" "example" {
+  name             = var.name
+  description      = var.description
+  enabled         = true
+  severities      = var.severities
+  resource_groups = var.resource_groups
+  email_alert_channels = var.channels
+
+  aws_compliance_reports {
+    pci = var.aws_pci
+    cis_s3 = true
+  }
+
+  gcp_compliance_reports {
+    pci = var.gcp_pci
+    cis = true
+  }
+
+  daily_compliance_reports {
+    aws_cloudtrail = var.daily_cloudtrail
+  }
+
+  weekly_snapshot = var.snapshot
+}
+
+variable "name" {
+  type    = string
+  default = "Report Rule 1"
```

Comment:
report rule name is not unique?

Reply:
It is not, but Report Rule 1 is not a good example name I will update this